SocketRocket.podspec

```diff
@@ -11,8 +11,10 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = '6.0'
   s.osx.deployment_target = '10.8'
+  s.tvos.deployment_target = '9.0'
```

Comment:
Is this right? Curious that tvos uses the corresponding iOS version number instead of it's own number. Would have expected this to be '1.0'